Build.fsx

```diff
@@ -17,7 +17,7 @@ type GitVersion = { apiVersion:string; nugetVersion:string }
 let getGitVersion = 
     let desc = Git.CommandHelper.runSimpleGitCommand "" "describe --tags --long --match=v*"
     // Example for regular: v3.50.2-288-g64fd5c5b, for prerelease: v3.50.2-alpha1-288-g64fd5c5b
-    let result = Regex.Match(desc, @"^v(?<maj>\d+)\.(?<min>\d+)\.(?<rev>\d+)(?<pre>-(alpha|beta|rc)\d*)?-(?<num>\d+)-g(?<sha>[a-z0-9]+)$", RegexOptions.IgnoreCase).Groups
+    let result = Regex.Match(desc, @"^v(?<maj>\d+)\.(?<min>\d+)\.(?<rev>\d+)(?<pre>-\w+\d*)?-(?<num>\d+)-g(?<sha>[a-z0-9]+)$", RegexOptions.IgnoreCase).Groups
```

Comment:
Again, it would be better if we didn't have to support all these use cases on our own. See my [previous comment](https://github.com/AutoFixture/AutoFixture/pull/749/commits/6e6dad941de349d9c538379d9d09b79cddbdb38d#r110601832).